src/containers/org/MemberRoleCard.js

```diff
@@ -55,32 +22,20 @@ const MemberRoleCard = ({
   role,
   isUnassignable,
 } :Props) => {
-
+  const dispatch = useDispatch();
   const roleId :UUID = role.id || '';
   const rolePath = ORG_ROLE
     .replace(ORG_ID_PARAM, organizationId)
     .replace(ROLE_ID_PARAM, roleId);
 
+  const goToRolePath = () => dispatch(goToRoute(rolePath));
+
   const handleClick = () => {
     onClick(role);
   };
 
   return (
-    <RoleCard>
-      <Flex>
-        <StyledRoleIcon />
-        <StyledLink to={rolePath}>
-          {role.title}
-        </StyledLink>
-      </Flex>
-      {
-        isUnassignable && (
-          <IconButton aria-label="unassign-role" onClick={handleClick}>
-            <FontAwesomeIcon fixedWidth icon={faTimes} />
-          </IconButton>
-        )
-      }
-    </RoleCard>
+    <Chip label={role.title} onClick={goToRolePath} onDelete={isUnassignable ? handleClick : undefined} />
```

Comment:
can you make this a proper link? Users should be able to middle click/open in a new tab. chance to drop dispatch as well.

```<Chip label={role.title} component="a" href={`#${rolePath}`} clickable onDelete={...} /> ```